### PR TITLE
Log Export Doc

### PR DESCRIPTION
**Pull request by lchapman4**
Added a quick blurb and screenshot of our new Export functionality in the Logs UI.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by lchapman4** [comment]
Ugh, my PR is unverified 😮‍💨. Hold please while I fix that.

**Comment by lchapman4** [comment]
Ok, GPG should be fixed. Last commit shows as verified.

**Review by ghost — DISMISSED** [review]

**Comment by lchapman4** [comment]
Just wanted to add this doc entry is for the Log Export feature announced here: https://docs.newrelic.com/whats-new/2023/08/whats-new-08-10-export-logs/

File: src/content/docs/logs/ui-data/use-logs-ui.mdx [hunk 1]
```diff
@@ -211,6 +213,16 @@ To explore your logging data, follow this basic workflow.
 
 </CollapserGroup>
 
+## Export [#export]
+
+To export a subset of your logs to share, users can leverage our dedicated Export feature in the Logs UI. This feature will allow an export of up to 2000 log records in either CSV or JSON format. The log records will be downloaded locally on your machine in the Downloads folder of your current browser.
```
Comment (ally-sassman) [inline src/content/docs/logs/ui-data/use-logs-ui.mdx:218]:
```suggestion
To export a subset of your logs to share, users can leverage our dedicated Export feature in the Logs UI. This feature will allow an export of up to 2000 log records in either CSV or JSON format. The log records will be downloaded locally on your machine in the `/Downloads/` folder of your current browser.
```

**Review by ghost — APPROVED** [review]

**Pull request merged**